Dockerfile.jetty

```diff
@@ -9,7 +9,7 @@ RUN mvn --batch-mode --define java.net.useSystemProxies=true package
 ########################################################################################
 
 FROM jetty
-MAINTAINER D.Ducatel
```

Comment:
MAINTAINER tag is deprecated.
https://docs.docker.com/engine/reference/builder/#maintainer-deprecated